test-apps/test-watchdog/src/main.cpp

```diff
@@ -1,12 +1,21 @@
 #include "WatchdogWrapper.h"
 #include "mbed.h"
 
+Thread pet_thread;
 std::chrono::milliseconds countdown_ms = 1000ms;
 std::chrono::milliseconds pet_ms       = 200ms;
 
+void pet_dog_task(std::chrono::milliseconds *pet_ms) {
+  while (1) {
+    Utility::WatchdogWrapper::petWatchdog();
```

Comment:
LMAO i've never heard of petting a watchdog. always seen kick :( TIL

Reply:
Kick is so mean though :(